.gitignore

```diff
@@ -22,3 +22,5 @@
 /phpunit.xml
 .phpunit.result.cache
 ###< phpunit/phpunit ###
+
+composer.lock
```

Comment:
composer.lock must be push :)

Reply:
it's a static file of installed app, when u deploy a new app u can hav lot of error like about the php version if different

Reply:
https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control